app/views/vms/edit.html.erb

```diff
@@ -22,6 +22,18 @@
       <%= form.text_area :description, cols: "40", rows: "4", class: 'form-control', value: @description %>
   </div>
 
+  <% if current_user.admin? %>
```

Comment:
Is there a reason why you do not change the FormBuilder to the new HartFormBuilder an use the the labeled_text_field?

Reply:
This issue is addressed right? The conversation can be resolved?

Reply:
Yes.